solutions/models.py

```diff
@@ -0,0 +1,37 @@
+from django.db import models
+from django_nanoid.models import NanoIDField 
+from utils.choices import Sender
+
+# Create your models here.
+
+class SolutionHistory(models.Model):
+    id = NanoIDField(
+    primary_key=True,
+    editable=False,
+    unique=True,
+    max_length=21,
+    )
+    user=models.ForeignKey('accounts.User', on_delete=models.CASCADE, related_name='solutionhistory')
+    mission=models.ForeignKey('missions.Mission', on_delete=models.CASCADE, related_name='solutionhistory')
+    created_at = models.DateTimeField(auto_now_add=True)
+    solved_at = models.DateTimeField(null=True, blank=True)
+    is_solved = models.BooleanField(default=False)
+
+    def __str__(self):
+        return f"{self.user} {self.mission.title} {self.created_at}"
+
+
+    
+class Message(models.Model):
+    solution_history = models.ForeignKey(SolutionHistory, on_delete=models.CASCADE, related_name='message')
+    created_at = models.DateTimeField(auto_now_add=True)
+    sender = models.CharField(
+        max_length=10,
+        choices=Sender.choices,
+        default=Sender.USER
+    )
```

Comment:
❗ ERD에서 IntegerField로 정했었습니다. 또, IntegerChoices는 IntegerField에만 사용할 수 있습니다.
```suggestion
    sender = models.IntegerField(
        choices=Sender.choices,
    )
```

Reply:
제가 수정내역을 다 반영하지 못했네용 죄송합니다ㅜ,ㅠ